app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(team: list) -> int:
```

Comment:
use more accurate annotation, list with what?

Reply:
i add add Elf in Dwarf insine list